README-zh.md

```diff
@@ -184,20 +275,21 @@ You can also reach the maintainers of this project at:
 
 你也可以通过以下渠道联系本项目的维护人员：
 
-- [Slack](https://kubernetes.slack.com/messages/sig-docs) [加入Slack](https://slack.k8s.io/)
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+  - [加入Slack](https://slack.k8s.io/)
```

Comment:
```suggestion
  - [获得此 Slack 的邀请](https://slack.k8s.io/)
```

Reply:
Thanks! I've force-pushed this better translation.